lib/editing/editor_codemirror.dart

```diff
@@ -179,6 +210,12 @@ class _CodeMirrorEditor extends Editor {
   set completionAutoInvoked(bool value)
       => cm.jsProxy['state']['completionAutoInvoked'] = value;
 
+  bool get lookingForQuickFix
```

Comment:
I think this setter and getter pair can move to being just a regular private field - `_lookingForQuickFix`.